docs/sql-migration-guide.md

```diff
@@ -45,6 +45,9 @@ license: |
     * `ALTER TABLE .. ADD PARTITION`
     * `ALTER TABLE .. RENAME PARTITION`
     * `ALTER TABLE .. DROP PARTITION`
+    * `ALTER TABLE .. RECOVER PARTITIONS`
+    * `MSCK REPAIR TABLE`
+    * `LOAD DATA`
```

Comment:
This doc change is for only `master`, right?

Reply:
@dongjoon-hyun This one, yes but we made some fixes in 3.0/3.1 regarding to cache refreshing:
- 3.0: #31006, #31060, #31236, #31116, #31305
- 3.1: #31006, #31060, #31235, #31115, #31304

Should I update the SQL migration guide about the changes, and say that "the commands clear cached table data"? cc @cloud-fan @HyukjinKwon

Reply:
In that case, for each branch, we may need a separate PR and independent update (if we want to update the doc).

Reply:
@HyukjinKwon @cloud-fan Do we want to update the SQL migration guide?

Reply:
"the commands clear cached table data" seems like a correctness bug fix that doesn't need a migration guide?

Reply:
I would expect that currently users have to uncache and cache tables before any table modifications like:
```
UNCACHE TABLE tbl;
ALTER TABLE tbl ...;
CACHE tbl;
```
After reading the SQL migration guide, they could remove the legacy code.